test/parallel/test-dgram-send-bad-arguments.js

```diff
@@ -28,17 +28,17 @@ const buf = Buffer.from('test');
 const host = '127.0.0.1';
 const sock = dgram.createSocket('udp4');
 
-assert.throws(function() {
```

Comment:
Since the function contains only one statement, the curly brackets are not required here and other places in the code.
That means, `() => { sock.send(); }` can be `() => sock.send()`

Reply:
I disagree that the brackets should be removed here. They should only be removed if the arrow function is explicitly returning a value, IMO.

Reply:
In other words:

`() => { return foo(); }` should be changed to `() => foo()` but `() => { foo(); }` should be left as-is, in my opinion. Otherwise, someone reading the code is left wondering if the fact that the arrow function is returning a value is a result of the author wanting to omit the brackets for the sake of omitting the brackets or if it's because the function is meant to return a value.